src/components/screens/ImportWalletScreen.tsx

```diff
@@ -47,6 +52,11 @@ export const ImportWalletScreen: React.FC<ImportWalletScreenProps> = ({
         value={recoveryPhrase}
         onChangeText={setRecoveryPhrase}
       />
+      {error && (
```

Comment:
should we use the built-in `error` prop from `Textarea` to render the error here?